student.rb

```diff
@@ -6,7 +6,13 @@ def initialize(age, classroom, name = 'unknown', parent_permission: true)
     @classroom = classroom
   end
 
+  attr_accessor :classroom
```

Comment:
- [ ] Classroom should be an `attr_reader` since the Student class already has a method of adding classrooms 👍